boa_engine/src/vm/opcode/value/mod.rs

```diff
@@ -31,3 +31,21 @@ impl Operation for ValueNotNullOrUndefined {
         Ok(CompletionType::Normal)
     }
 }
+
+/// `IsObject` implements the Opcode Operation for `Opcode::IsObject`
+///
+/// Operation:
+///  - Require the stack value to be neither null nor undefined.
```

Comment:
```suggestion
///  - Pushes `true` if the top stack value is an object, `false` otherwise.
```
Shouldn't is say that is checks if it is an object?